### PR TITLE
Add New Example (Simple Memo CRUD)

### PR DESCRIPTION
**Pull request by Hazealign**
Made New Example for Simple Memo CRUD. (With mongoose.)

### CODE REVIEW

File: examples/memo/index.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  });
+
+  memo.save();
+  return memo;
+});
+
+app.patch("/memo/:memoId", function *(req, res) {
+	var memo = yield Memo.findById(new ObjectId(req.params.memoId));
+
+  if (memo === undefined || memo === null) {
```
Comment (therne) [inline examples/memo/index.js]: I think it can be simplified to `if (!memo)`

File: examples/memo/index.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    return;
+  }
+
+  let length = parseInt(req.body.length);
+	let offset = (parseInt(req.body.pageNo) - 1) * length;
+	return yield Memo.find({}, null, { skip: offset, limit: length });
+});
+
+app.get("/memo/:memoId", function *(req, res) {
+	return yield Memo.findById(new ObjectId(req.params.memoId));
```
Comment (therne) [inline examples/memo/index.js]: `new ObjectId` is redundant because mongo converts `string` to `ObjectId` automatically

**Comment by therne** [comment]
Looks good but here are convention problems, too 😟 

- Tabs and spaces are mixed
- Use `const` instead of `let` in module importing
- There are some redundant codes... we need to keep it clear because it's example code :)

**Comment by Hazealign** [comment]
I'm sorry for couldn't check my editor's coding conventions.
I change tab to 4 spaces, and use const instead of let in module importing.

Check plz. Thanks. :)

File: examples/memo/index.js [hunk 3]
```diff
@@ -15,25 +14,29 @@ let app = cottage({ caseSensitive: true });
...
 
 function findProperties (array, object) {
-	var returnVal = true;
-	for (let val of array) if (!object.hasOwnProperty(val)) returnVal = false;
+    var returnVal = true;
+    for (let val of array) if (!object.hasOwnProperty(val)) returnVal = false;
     return returnVal;
 }
 
+function checkNullOrUndefined (obj) {
```
Comment (therne) [inline examples/memo/index.js]: Can be simplified to `obj == null`

**Pull request merged**